mobify/progressive-web-scaffold
# Custom header/footer support (checkout feature)

Most of the checkout flow has a different header and footer (according to the mocks) The scaffold currently does not support overriding/customizing this, so this PR addresses this. 

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1015

## Changes
- Allow routes to optionally specify a header and/or footer to use
- Implement a basic header/footer for the checkout

## How to test-drive this PR
- Preview and observe the header/footer throughout the site is unchanged
- Go to https://www.merlinspotions.com/checkout/shipping/ and observe the header and footer look different
